pynumaflow/function/_dtypes.py

```diff
@@ -28,17 +26,27 @@ class Message(metaclass=NoPublicConstructor):
     """
 
     _keys: List[str]
+    _tags: List[str]
     _value: bytes = b""
 
-    @classmethod
-    def to_vtx(cls: Type[M], keys: List[str], value: bytes) -> M:
+    def __init__(self, value: bytes):
```

Comment:
@vigith any reason why we are not allowing users to initialize the object from the constructor itself?

Reply:
discussed with Vigith offline, and decided to go with constructor for object creation.